www/source/blog/2018-06-13-base-plans-refresh.html.md

```diff
@@ -0,0 +1,170 @@
+   widget_world: WARN
+   widget_world: WARN The current situation usually arises when a Plan has a direct
+   widget_world: WARN dependency on one version of a package (`acme/A/7.0/20160101200001`)
+   widget_world: WARN and has a direct dependency on another package which itself depends
+   widget_world: WARN on another version of the same package (`acme/A/2.0/20151201060001`).
+   widget_world: WARN If this package (`acme/A`) contains shared libraries which are
+   widget_world: WARN loaded at runtime by the current Plan, then both versions of
+   widget_world: WARN `acme/A` could be loaded into the same process in a potentially
+   widget_world: WARN surprising order. Worse, if both versions of `acme/A` are
+   widget_world: WARN ABI-incompatible, runtime segmentation faults are more than likely.
+   widget_world: WARN
+   widget_world: WARN In order to preserve reliability at runtime the duplicate dependency
+   widget_world: WARN entries will need to be resolved before this Plan can be built.
+   widget_world: WARN Below is an expanded graph of all `$pkg_deps` and their dependencies
+   widget_world: WARN with the problematic lines noted.
+   widget_world: WARN
+   widget_world: WARN Computed dependency graph (Lines with '*' denote a problematic entry):
+
+nellshamrell/widget_world/1.0.0/<timestamp>
+    nellshamrell/widget/1.0.0/<timestamp> (*)
+        core/glibc/2.27/<timestamp> (*)
+    nellshamrell/world/1.0.0/<timestamp> (*)
+        core/glibc/2.22/<timestamp> (*)
+
+ERROR: Computed runtime dependency check failed, aborting
+```
+
+This error occurs because nellshamrell/widget_world has one dependency that was built with 2.27, and another that was built with 2.22.  This makes them incompatible, and the nellshamrell/widget_world Plan will not build.
+
+### How do I resolve this?
+
+In this case, you can resolve this error by rebuilding nellshamrell/world with core/glibc version 2.27.  If we do that, and now both dependencies depend on the same version of glibc, then nellshamrell/widget_world will build fine.
+
+```
+nellshamrell/widget_world
+-> nellshamrell/widget
+   --> core/glibc/2.27
+-> nellshamrell/world
+   --> core/glibc/2.27
+```
+
+Success!
+
+```
+(studio) $ build widget_world
+...
+Success!
+I love when a Plan.sh comes together!
+```
+
+So...after the Core Plans refresh gets added to production and promoted to stable (which, again, we are doing on Tuesday, June 19th), and all of the Base Plans and dependent Base Plans are promoted to stable, you may see some of these failures with your own Plans.  It's also possible they may fail on more than one dependency being out of sync.  You can fix these errors in 2 ways:
+1) (This is the option we recommend!) Rebuilding a dependency with an older version of a Base Plan (i.e. nellshamrell/world with core/glibc/2.22) with a newer version of that Base Plan, or at least the same version as the rest of your dependencies (i.e. nellshamrell/world with core/glibc/2.27).
```

Comment:
Need an empty line here also in order to get the numbering to lay out properly, and then you can use dots rather than parens:

```
1. (This is...)

1. Pinning...
```

That'll make the formatting happen properly. (You can always run this locally, by the way, by running `make run` from within the `www` directory.)